.travis.yml

```diff
@@ -1,13 +1,23 @@
 language: ruby
 
+git:
+  depth: false
+
 before_install:
   - gem install bundler -v '<2'
 
 rvm:
   - 2.3
   - 2.7
 
-env: CI="travis"
+env:
+  global:
+    - CI="travis"
+
+jobs:
+  include:
+    - arch: s390x
+      rvm: 2.7.1
```

Comment:
How about changing this part `rvm: 2.7.1` to `rvm: 2.7` to run on the latest Ruby 2.7? It's good to run on Ruby 2.7.X in the future.

Reply:
Good call! I'll fix that.